packages/@aws-cdk/aws-redshift-alpha/lib/cluster.ts

```diff
@@ -299,10 +299,17 @@ export interface ClusterProps {
   readonly masterUser: Login;
 
   /**
-   * A list of AWS Identity and Access Management (IAM) role that can be used by the cluster to access other AWS services.
+   * A default AWS Identity and Access Management (IAM) role to be used by the cluster to access other AWS services.
+   *
+   * @default - Create a new role
+   */
+  readonly serviceRole?: iam.IRole;
+
+  /**
+   * A list of additional AWS Identity and Access Management (IAM) roles that can be used by the cluster to access other AWS services.
```

Comment:
Somewhat conflicted on whether this should even be a prop, but I am leaning towards having it since other services follow a similar pattern.